### PR TITLE
add support for special links

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

**Comment by tzarebczan** [comment]
Can you add one for ?discover also - a popular one we use on the app just so we can get people to link a link/open the app automatically.

**Comment by akinwale** [comment]
It defaults to discover if it doesn't match any of the following:
* verify
* rewards
* settings
* trending
* wallet

**Comment by tzarebczan** [comment]
Perfect!

File: app/src/utils/helper.js [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+function getRouteForSpecialUri(uri) {
+  let targetRoute;
+  const page = uri.substring(8).trim(); // 'lbry://?'.length == 8
+
+  switch (page) {
+    case 'rewards': targetRoute = 'Rewards'; break;
+    case 'settings': targetRoute = 'Settings'; break;
+    case 'trending': targetRoute = 'TrendingStack'; break;
+    case 'wallet': targetRoute = 'WalletStack'; break;
+    default: targetRoute = 'DiscoverStack'; break;
```
Comment (seanyesmunt) [inline app/src/utils/helper.js]: These should probably be constants.

Comment (akinwale) [reply app/src/utils/helper.js]: Agreed!

**Pull request merged**